File: 1. Learn the Basics/2. Basics Of Maths/C++/1. Count Digits.cpp
/*
Count number of digits in a number
*/

#include <bits/stdc++.h>
using namespace std;

int countNumber(int n) {
    // Write your code here.
    int count = 0;
    while(n>0){
        n = n/10;
        count++;
    }
    return count;
}


int main() {
	// your code goes here
	int n;
	cin >> n;
	int ans = countNumber(n);
	cout << ans;
	return 0;
}

File: 1. Learn the Basics/2. Basics Of Maths/Java/1. Count Digits.java
/* package codechef; // don't place package name! */

/*
Count number of digits in a number
*/
import java.util.*;
import java.lang.*;
import java.io.*;


/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
    public int countDigits(int n) {
        // Write your code here.
        int count = 0;
        while(n > 0){
            n /= 10;
            count++;
        }
        return count;
    }
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Codechef obj = new Codechef();
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int ans = obj.countDigits(n);
		System.out.println(ans);
	}
}
